source/manage/common-support-packet.rst

```diff
@@ -5,15 +5,14 @@ Use the System Console or the :ref:`mmctl system supportpacket <manage/mmctl-com
 Contents of a support packet
 ----------------------------
 
-A Mattermost Support Packet can contain the following files:
+A Mattermost support packet can contain the following files:
```

Comment:
I think it makes sense to use capital letters for the term "Support Packet" as it's a special term used only in Mattermost. It also matches the casing in the UI and other documents.